test/integration/scheduler/scheduler_test.go

```diff
@@ -485,7 +484,7 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			framework.ExpectedResourceFlavorToBeDeleted(ctx, k8sClient, onDemandFlavor, true)
 		})
 
-		ginkgo.It("Should schedule jobs from the selected namespaces", func() {
+		ginkgo.It("Should schedule jobs from the selected namespaces after label update", func() {
```

Comment:
We need a better test case where workloads in another namespace stay in inadmissible state. But maybe we can do that in unit tests

Reply:
added to this test